pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -90,6 +90,54 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *v1.Prometheu
 			{
 				Name: "kubevirt.rules",
 				Rules: []v1.Rule{
+					{
+						Record: "virt_controller_rest_client_requests",
+						Expr: intstr.FromString(
+							fmt.Sprintf("rest_client_requests_total{pod=~'virt-controller-.*', namespace='%s'}", ns),
```

Comment:
to be honest, I don't understand why we need the namespace filter: `namespace='%s'`

Reply:
yes exactly(we used this filter before my modifications as well)